Cargo.toml

```diff
@@ -34,9 +34,7 @@ members = [
 ]
 
 # Due to --cfg=web_sys_unstable_apis
-exclude = [
```

Comment:
Please split formatting changes into a separate PR before this one.

It makes it difficult to review your changes.